src/library-authoring/list-libraries/specs/LibraryListPage.spec.jsx

```diff
@@ -30,6 +41,38 @@ describe('list-libraries/LibraryListPage.jsx', () => {
     );
   });
 
+  it('renders library list page with loading', () => {
+    const newProps = update(props, {
+      status: { $set: LOADING_STATUS.LOADING },
+    });
+
+    const container = ctxMount(
+      <BrowserRouter>
+        <InjectedLibraryListPage {...newProps} />
+      </BrowserRouter>,
+    );
+
+    const loading = container.find('.spinner-border').at(0);
```

Comment:
It might be simplified to `const loading = container.find('.spinner-border');`

Reply:
done